devashish_170102026.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib.pyplot as plt
+c='data.txt'
```

Comment:
Not relevant here since it is a small code, but it's generally advised to use proper variable names for interpretability and code readability. E.g. instead of `c`, it would be better to use `file_name`.